docs/content/get-started/installation/controller.md

```diff
@@ -168,6 +168,15 @@ kubectl get controller -A
 You should see pods for Aperture Controller and Controller Manager in `RUNNING`
 state and `Controller` Custom Resource in `created` state.
 
+## Applying Policies
+
+Follow the information on [Policy](/concepts/policy/circuit/sources.md) to
```

Comment:
How do we take the blueprint policy and add it to the ConfigMap?

Reply:
We have the steps for the same under the blueprint [docs](https://docs.fluxninja.com/docs/development/get-started/blueprints#using-aperture-blueprints-as-a-jsonnet-mixins-library)

Reply:
Got it